services/vorbis_codec_configuration_service.go

```diff
@@ -0,0 +1,96 @@
+package services
+
+import (
+	"encoding/json"
+
+	"github.com/bitmovin/bitmovin-go/bitmovin"
+	"github.com/bitmovin/bitmovin-go/models"
+)
+
+type VorbisCodecConfigurationService struct {
+	RestService *RestService
+}
+
+const (
+	// FIXME Double check endpoint
+	VorbisCodecConfigurationEndpoint string = "encoding/configurations/audio/vorbis"
+)
+
+func NewVorbisCodecConfigurationService(bitmovin *bitmovin.Bitmovin) *VorbisCodecConfigurationService {
+	r := NewRestService(bitmovin)
+
+	return &VorbisCodecConfigurationService{RestService: r}
+}
+
+func (s *VorbisCodecConfigurationService) Create(a *models.VorbisCodecConfiguration) (*models.VorbisCodecConfigurationResponse, error) {
```

Comment:
Same here with naming, but I know, that's kind of go style

Reply:
it would be more go style if it's called `c` not `a` ;)
I'd also name this `conf` - still short enough but more readable..